app/main/views/services.py

```diff
@@ -419,6 +419,9 @@ def edit_service_submission(framework_slug, lot_slug, service_id, section_id, qu
         framework=framework,
         service_data=draft,
         service_id=service_id,
+        one_service_limit=[
+            l['oneServiceLimit'] for l in framework['lots'] if l['slug'] == draft['lot']
+            ][0],
```

Comment:
OK I'll sort that out.  It was a pure copy-and-paste job from elsewhere that did the same thing I'm afraid.